pkg/dss/scd/cockroach/operations.go

```diff
@@ -0,0 +1,31 @@
+package cockroach
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/golang/geo/s2"
+	dssmodels "github.com/interuss/dss/pkg/dss/models"
+	scdmodels "github.com/interuss/dss/pkg/dss/scd/models"
+)
+
+var (
+	errNotImplemented = errors.New("not implemented")
+)
+
+func (s *Store) GetOperation(ctx context.Context, id scdmodels.ID) (*scdmodels.Operation, error) {
+	return nil, errNotImplemented
+}
+
+func (s *Store) DeleteOperation(ctx context.Context, id scdmodels.ID, owner dssmodels.Owner) (*scdmodels.Operation, []*scdmodels.Subscription, error) {
```

Comment:
Delete should take a version as well

Reply:
I looked through the API spec, and didn't see the version in the request parameters, hence why I didn't add it to the store.

Reply:
@BenjaminPelletier  we should be including the version here for deletions